job-service/src/main/java/com/hpe/caf/services/job/jobtype/JsltTaskBuilder.java

```diff
@@ -58,12 +60,14 @@
     public JsltTaskBuilder(
         final String jobTypeId,
         final Map<String, String> configuration,
+        final Map<String, String> constants,
         final ParametersValidator parametersValidator,
         final String taskScript
     ) throws InvalidJobTypeDefinitionException {
         this.jobTypeId = jobTypeId;
         this.configuration = configuration;
         this.parametersValidator = parametersValidator;
+        this.constants = constants;
```

Comment:
Keep to initialize variables in the same order that you've listed the instance variables.

```suggestion
        this.constants = constants;
        this.parametersValidator = parametersValidator;
```